src/Suave/Sockets/ITransport.fs

```diff
@@ -1,5 +1,7 @@
 ﻿namespace Suave.Sockets
 
+[<AllowNullLiteral>]
```

Comment:
Should we add a comment to this effect on ITransport; F# devs may not be expecting nulls, and usage of this interface goes all the way up to the user during configuration.